docker/create_dockerfile_and_build.py

```diff
@@ -199,7 +198,7 @@ def build_docker_image(ddir, dockerfile_name, container_name):
         FLAGS.image_name = "tritonserver_with_ft"
 
     if FLAGS.base_image is None:
-        base_image = "nvcr.io/nvidia/tritonserver:" + FLAGS.triton_version + "-py3"
+        base_image = "nvcr.io/nvidia/tritonserver:" + FLAGS.triton_version + "-py3-min"
```

Comment:
I don't think we need to change the default base image. If we wan tot build with base image, we can do:
`python3 docker/create_dockerfile_and_build.py --triton-version=22.12 --base-image "nvcr.io/nvidia/tritonserver:22.12-py3-min --image-name tritonserver_22.12_with_ft`